test-data/unit/check-basic.test

```diff
@@ -261,6 +261,16 @@ main:3: error: Incompatible types in assignment (expression has type "B", variab
 main:7: error: Incompatible types in assignment (expression has type "A", variable has type "B")
 main:9: error: Incompatible types in assignment (expression has type "B", variable has type "A")
 
+[case testUnderscoreVariableIsFresh]
+_ = 1
+_ = ""
+_ = ""  # type: int
+(_, _) = ("", 1.0 + "")  # E: Unsupported left operand type for + ("float")
+(_, _) = (1.0, "")
+for (_, _) in ((1, "a"), (2, "b")): pass
+
+def f(_: int, _: str) -> None: pass
```

Comment:
This actually gives ``SyntaxError: duplicate argument '_' in function definition`` at runtime (also in Python 2 with annotations removed).